File: examples/emoji/send/go/sendATextContainingAnEmojiToAnyoneViaSms.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/emoji"
)

// Send an emoji to anyone via SMS. Messages are sent in the form '<message> Sent from <from>'
func SendAtextContainingAnEmojiToAnyoneViaSms() {
	emojiService := emoji.NewEmojiService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := emojiService.Send(&emoji.SendRequest{
		From:    "Alice",
		Message: "let's grab a :beer:",
		To:      "+44782669123",
	})
	fmt.Println(rsp, err)
}

File: examples/js/emoji/send/sendATextContainingAnEmojiToAnyoneViaSms.js
const { EmojiService } = require("m3o/emoji");

// Send an emoji to anyone via SMS. Messages are sent in the form '<message> Sent from <from>'
async function sendAtextContainingAnEmojiToAnyoneViaSms() {
  let emojiService = new EmojiService(process.env.MICRO_API_TOKEN);
  let rsp = await emojiService.send({
    from: "Alice",
    message: "let's grab a :beer:",
    to: "+44782669123",
  });
  console.log(rsp);
}

sendAtextContainingAnEmojiToAnyoneViaSms();
